src/actions.js

```diff
@@ -23,6 +22,14 @@ export const handleFileLoading = (e) => {
   }
 }
 
+export const handleVideoSelection = e => {
+  return dispatch => {
+
+    let file = e.target.files[0]
```

Comment:
Aqui você pode colocar também 
```suggestion
    const [ file ] = e.target.files
```

Reply:
Yeah much better, thanks @igorgcosta!